pkg/storage/cacher.go

```diff
@@ -405,10 +408,15 @@ func (c *cacheWatcher) add(event watchCacheEvent) {
 }
 
 func (c *cacheWatcher) sendWatchCacheEvent(event watchCacheEvent) {
-	curObjPasses := event.Type != watch.Deleted && c.filter(event.Object)
+	curObjPasses := false
```

Comment:
Hrm - so before, this didn't execute filter if the watch type was deleted.  Now it does.  Is that an intentional change, and does it match the regular watcher behavior?